jnigen/test/simple_package_test/runtime_test_registrant.dart

```diff
@@ -528,6 +529,74 @@ void registerTests(String groupName, TestRunnerCallback test) {
       });
     });
   });
+
+  group('interface implementation', () {
+    for (final method in {
+      'another thread': MyInterfaceConsumer.consumeOnAnotherThread,
```

Comment:
It would be nicer to use records instead of a map here, it's easier to read on the callsite. However, that means forcing the SDK lower bound to 3.0.

Reply:
I'll make a slightly bigger change to the codebase after upgrading to 3.0.0 in a follow-up PR.